packages/fluentui/react-northstar/src/themes/teams/components/Card/cardVariables.ts

```diff
@@ -36,6 +41,11 @@ export default (siteVars): CardVariables => {
     borderColor: siteVars.colors.black,
     borderWidth: siteVars.borderWidth,
     borderRadius: pxToRem(4),
+    boxShadow: undefined,
+    boxShadowHover: siteVars.shadowLevel3,
+    boxShadowPressed: undefined,
+    boxShadowSelected: undefined,
+    boxShadowDisabled: undefined,
```

Comment:
Cruft? I don't that these variables are used anywhere

Reply:
Oops, good catch, will remove :)